assets/app/view/game/entities.rb

```diff
@@ -47,13 +47,19 @@ def render
           h(Bank, game: @game),
           h(GameInfo, game: @game, layout: 'upcoming_trains'),
           *@game.corporations.select(&:receivership?).map { |c| h(Corporation, corporation: c) },
+          render_unsold_companies,
```

Comment:
Given that 1860 should probably show the unowned company if the bank owns this 'unsold' is incorrect and probably should be generalized.

render_unowned_companies would be better

Reply:
done